main/FatesHistoryInterfaceMod.F90

```diff
@@ -4048,16 +4057,6 @@ subroutine define_history_vars(this, initialize_variables)
           avgflag='A', vtype=site_size_r8, hlms='CLM:ALM', flushval=0.0_r8,    &
           upfreq=1, ivar=ivar, initialize=initialize_variables, index = ih_carbon_balance_understory_si_scls )
     
-    call this%set_history_var(vname='DBALIVEDT_CANOPY_SCLS', units = 'kg C / ha / yr',               &
```

Comment:
just want to confirm that we're getting rid of these variables because they're too esoteric? that's fine, but want to check that that is the rationale.